package.json

```diff
@@ -2,7 +2,7 @@
   "name": "gather",
   "displayName": "Gather",
   "description": "Adds gather functionality to the Python Extension",
```

Comment:
Looks like we might need a rename here. Or...do we? It adds it to the Jupyter extension, but you also need the Python extension to gather.

Reply:
Yeah, that's a weird one, it works without the python extension, but only supports the python language.